app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
Ok, thanks! It turns out the case where a user does not select a locale causes another infinite re-direct where the beforeModel hook gets run repeatedly, so that case will need to be handled differently.